File: algorithm_study/xinghuaOj/1039.py
n = int(input())
cityNum = ["A", "B", "C", "D", "E", "F", "G", "R", "S", "T"]
count = n
for i in range(0, n):
    carNum = input()
    for it in cityNum:
        if carNum[0] == it:
            count -= 1
            break

print(count)

File: algorithm_study/test/1039.cpp
#include <array>
#include <iostream>
#include <string>

using namespace std;
int main() {
  std::array<string, 10> cityNum = {"A", "B", "C", "D", "E",
                                    "F", "G", "R", "S", "T"};
  int n = 0;
  string carNum;
  cin >> n;
  int count = n;
  for (int i = 0; i < n; i++) {
    cin >> carNum;
    carNum.erase(1, 5);
    for (auto it : cityNum) {
      if (it == carNum) {
        count--;
        break;
      }
    }
  }
  cout << count;
}